generators/server/templates/quarkus/src/main/java/package/web/rest/vm/LoginVM.java.ejs

```diff
@@ -0,0 +1,47 @@
+<%#
```

Comment:
Sorry we use flyway and not jhipster liquibase, so it is new.

Reply:
👍

Reply:
It's not the same template.
JHipster use getter and setter where Quarkus is pushing to make the the fields public.

From a functionnal stand point it's similar but it's part of the Quarkus mind switch.

Reply:
Yes, you are right for the security topic... my advice was to maximize the reusability of jhipster templates and code, in order to avoid possible error code and to create new tests or coverage. Obviously where is possible to reuse code :)